harness/determined/common/util.py

```diff
@@ -48,6 +48,14 @@ def get_default_master_address() -> str:
     return os.environ.get("DET_MASTER", os.environ.get("DET_MASTER_ADDR", "localhost:8080"))
 
 
+def get_container_user() -> Optional[str]:
+    return os.environ.get("DET_USER")
+
+
+def get_user_token() -> Optional[str]:
```

Comment:
These names are better than before.

Nit: maybe `get_container_user_name` and `get_container_user_token`?  To show that they are collecting information from the same source, so to speak.